examples/tutorials/core_api_pytorch_mnist/model_def_metrics.py

```diff
@@ -193,7 +193,7 @@ def main(core_context):
     optimizer = optim.Adadelta(model.parameters(), lr=args.lr)
 
     scheduler = StepLR(optimizer, step_size=1, gamma=args.gamma)
-    for epoch in range(1, args.epochs + 1):
+    for epoch in range(0, args.epochs):
```

Comment:
I recommend changing from `epoch` to `epoch_idx` here, for clarity.  Also, up to you, but you might consider leaving a `# NEW` comment explaining the reason why we want the extra clarity of `epoch_idx`.